docs/diagrams/ViewState2.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
+!include style.puml
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+skinparam ClassBackgroundColor #FFFFAA
+
+title After command "view 2"
+
+package List {
+    class Person1 as "<u>p0:Person</u>"
+    class Person2 as "<u>p1:Person</u>"
+    class Person3 as "<u>p2:Person</u>"
+}
```

Comment:
I know that this issue arises due to the user inputs being one-indexed and the internal list being zero-indexed, but maybe its better that these diagrams are labelled as one-indexed as well (e.g. p1, p2, p3 instead of  p0, p1, p2). It feels weird that view 2 shows p1 although technically that is accurate.